doozer/doozerlib/metadata.py

```diff
@@ -284,21 +284,23 @@ def hotfix_brew_tags(self):
         """
         return [self.hotfix_brew_tag()]
 
-    def get_arches(self):
+    def get_arches(self, konflux: bool = False):
```

Comment:
Guess we no longer need to explicitly pass this argument. `runtime.build_system` can be used to determine if the current build system is konflux or not.

Reply:
@vfreex looks like there was a small bug, we weren't initializing the build system for `KonfluxBuildCli` till now.

Pushed the fix to this PR: https://github.com/openshift-eng/art-tools/pull/1410/commits/44c51b8b5ec5f6f2bc58cf6565b5cbad65d4e423

Reply:
this is surprising, I suggest we add the `--build-system` to the Doozer base command [here](https://github.com/openshift-eng/art-tools/blob/main/pyartcd/pyartcd/pipelines/ocp4_konflux.py#L71)

Reply:
Good idea, updating now

Reply:
updated: https://github.com/openshift-eng/art-tools/pull/1410/commits/fc233d6e40a727d4056357717dc498fa047c17e4 @locriandev

Reply:
hmm wait that doesn't seem to be working

Reply:
ahh nvm, I was testing the doozer command directly. its working